README.md

```diff
@@ -554,6 +554,43 @@ JSON formatted Zone Report
 }
 ```
 
+### /logicalpath/replicate
+Replicates a data object into some resource
+
+**Method**: POST
+
+**Parameters**:
+- all: If selected, updates all stale copies.
+- recursive:  Required to replicate a collection. Replicates the whole subtree. Does nothing if used on a data object.
+- thread-count: The number of threads to use for replication.
+- replica-number: Specifies the particular replicate to copy from. Typically not needed.
+- dst-resource: Resource to which to replicate the data object or collection.
+- logical-path: The logical path of the data object or collection to replicate.
+
+**Example CURL command**:
+```
+export TARGET_PATH=/tempZone/home/rods/hello.cpp
+curl -X POST -H "Authorization: ${TOKEN}" 'http://localhost/irods-rest/0.9.1/logicalpath/replicate?logical-path=$TARGET_PATH'
```

Comment:
0.9.2?

Reply:
Fixed in 1a4042d4b5f2aa35c9e5d11bacfbcfd0374b3221